scripts/bygg

```diff
@@ -12,17 +12,50 @@ SCRIPT_FOLDER=$(dirname "$SCRIPTNAME")
 PLATFORM_FOLDER=$(realpath "$(dirname "$SCRIPT_FOLDER")")
 
 # Parse options or show usage which exits app
-usage()
-{
-  echo "$0 - build bitbake recipes/images using docker"
-  echo "Usage: $(basename "$0" ) [options] build-command-to-run"
-  echo "options(: means required argument, :: means optional argument)"
-  for option in $(echo "$LONGOPTIONS" | tr , \\n); do
-    echo "--$option"
-  done
+usage() {
+
+  cat << EOF
+$0 - build bitbake recipes/images using docker
+Usage: $(basename "$0") [options] build-command-to-run
+options(: means required argument, :: means optional argument)
+
+OPTIONS:
+  -b, --build-folder     Optional, specify the folder for the build output. Default is 'build' directory.
+  -c, --delete-conf      Optional, delete existing configuration files in build-folder. Recomended to use if built diffrent machines and multiple machines.
+  -d, --distro           Optional, specify the target poky distro. Example are poky and fslc-wayland. Default is poky.
+  -f, --manifest-file    Required, specify the manifest file to use. See list of manifest files in this repo. Example kirkstone.xml.
+  -g, --skip-docker-pull Optional,  Skip update the docker container.
+  -h, --help             Display this help message and exit
+  -i, --image            Specify the bsp image to use for the build.
+                         Can be combine with recipe option or skiped if 'recipe' option is used.
+                         Multible images is supported (--image image1 --image image2).
+  -m, --machine          Specify the target machine for the build. Multible machine is supported.
+  -n, --skip-init-build  Optional, Setup bitbake, leaves us in build folder.
+  -o, --populate-sdk     Optional, Populate the SDK after the build.
+  -p, --package-server   Optional, Specify the packages server URL or local location. Will deploy directory path /packages/poky/manifest-in-use/ with ipk packages.
```

Comment:
I suggest renaming long option to --package-source, because a local directory is not a server.

local location -> local directory

I also don't quite understand the second sentence. Does it refer to the local directory? If so, and "/tmp/" is used as the argument, will the packages end up in "/tmp/packages/poky/manifest-in-use/"?